stencil/src/components/pulumi-api-doc-nav-node/pulumi-api-doc-nav-node.tsx

```diff
@@ -0,0 +1,135 @@
+import { Component, h, Listen, Prop } from "@stencil/core";
```

Comment:
Can't seem to put this comment on the file itself since it is empty, but appears the associated `.scss` file can be deleted?

Reply:
I think it is reasonable to leave have a scss file when setting up a component jsut for ease of adding styles in the future, but I have deleted it. cc @kimberleyamackenzie not sure what the norm is here